specs/Theme.md

```diff
@@ -0,0 +1,130 @@
+Theme API
+===
+
+# Background
+This API's main use is to set the overall theme for WebView2's which are associated with a profile. The options are similar to Edge: match the system default theme, change to light theme, or change to dark theme.
+This sets the theme for WebView2 UI like dialogs, prompts, and context menus by setting the ['prefers-color-scheme'](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features). Websites typically query for this setting in order to set CSS themes for light/dark.
```

Comment:
Given that

* CSS uses "color scheme" to describe what this API controls.
* Edge uses "overall appearance" to describe what this API controls.
* Edge uses the word "theme" to describe something different.

Maybe we should avoid the word "theme" since it doesn't align with *anything*.

Possible names: ColorScheme, PreferredColorScheme.

Reply:
+1 that "theme" doesn't fit very well. This wraps CSS `prefers-color-scheme`, so `ColorScheme` would be better, or even `PreferredColorScheme`.

Reply:
I agree. I think `PreferredColorScheme` would be most clear.